frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -157,6 +160,10 @@ class AccountsSummary extends Component<Props, State> {
 
   private async onStatusChanged(code: string, initial: boolean = false) {
     const status = await accountApi.getStatus(code);
+    const { offlineErrors } = this.state;
+    if (status.offlineError && !offlineErrors.includes(status.offlineError)) {
+      this.setState({ offlineErrors: [...offlineErrors, status.offlineError] });
```

Comment:
could also push `status.fatalError && t('account.fatalError')` into this array, or replace the array completely if 1 account had fatalError.

On the other hand the offlineError might contain useful information like Tor misconfiguration.

Reply:
I would not replace the array in case of fatalError, I agree with you that offlineError could be useful

Reply:
what if you get online again? this code seems to never reset offlineErrors when back online.